src/modules/new-request-form/NewRequestForm.tsx

```diff
@@ -47,11 +47,13 @@ export function NewRequestForm({
       onSubmit={handleSubmit}
     >
       {errors && <Alert type="error">{errors}</Alert>}
-      <TicketFormField
-        label={ticket_forms_instructions}
-        ticketFormField={ticket_form_field}
-        ticketForms={ticketForms}
-      />
+      {ticketForms.length > 0 && (
```

Comment:
This is already handled in the `TicketFormField` component: https://github.com/zendesk/copenhagen_theme/blob/b80ed7e9b4490d85f7ac1438268a29e4e8f7ebd4/src/modules/new-request-form/ticket-form-field/TicketFormField.tsx#L44
But maybe it makes sense to move it here for clarity (but we need to remove the other check)

Side note: is it possible to have 0 ticket forms? I added the check `ticketForms.length > 1` because if I don't configure the ticket forms I always have at least one (the default one)

Reply:
> is it possible to have 0 ticket forms?

Yup! When the plan does not have the feature, ticket forms is empty.
So basically we have this state:

- Customer **does not have**  the `ticket_forms` feature:
  - `ticketForms` is empty
  - We cannot post `request[ticket_form_id]` or validation will fail
- Customer **has**  the `ticket_forms` feature:
  - There is at least one `ticketForm`
  - We must post `request[ticket_form_id]` or validation will fail
    - If there's only one ticket form we don't want to show the dropdown
    - If there's several ticket forms we must show the dropdown

> This is already handled in the TicketFormField component:

It is for the dropdown but not for the hidden input field. So when there are no ticket forms, we're posting `request[ticket_form_id]` with the value of `null` and that fails validation:
https://github.com/zendesk/copenhagen_theme/blob/b80ed7e9b4490d85f7ac1438268a29e4e8f7ebd4/src/modules/new-request-form/ticket-form-field/TicketFormField.tsx#L42

> But maybe it makes sense to move it here for clarity (but we need to remove the other check)

I think we still need this check because we don't want to show the dropdown with only one option.

Reply:
Ah! I didn't know about it! It makes sense then.
 
Maybe I would add a comment stating that if the user has no ticket forms we shouldn't populate the `request[ticket_form_id]`, to make it clear why we have 2 different `if`s, one checking for 0 and another checking for 1